src/Restforce.php

```diff
@@ -53,28 +82,169 @@ public function __construct(
         $this->password = $password;
     }
 
-    public function create(string $sobjectType, array $data): ResponseInterface
+    /**
+     * Create method
+     *
+     * @param string $sobjectType object type
+     * @param array  $data        data
+     *
+     * @return mixed
+     */
+    public function create(string $sobjectType, array $data)
     {
         $uri = 'sobjects/' . $sobjectType;
 
         return $this->getOAuthRestClient()->postJson($uri, $data);
     }
 
-    public function update(string $sobjectType, string $sobjectId, array $data): ResponseInterface
+    /**
+     * Create bulk job
+     *
+     * @param string $object    object to work with
+     * @param string $operation operation
+     *
+     * @return string
+     * @throws Exception
+     */
+    public function createJob(string $object, string $operation)
```

Comment:
create.  BulkJob ?    Job => BulkJob

Reply:
It implies modifying these too:

createBulkJob
addBatchToBulkJob
executeBulkJob
checkBulkJob
closeBulkJob